tests/rare/RedwoodCorrectnessBTree.toml

```diff
@@ -7,3 +7,5 @@ startDelay = 0
     testName = 'UnitTests'
     maxTestCases = 1
     testsMatching = 'Lredwood/correctness/btree'
+    maxRunTimeWallTime = 1680.0
+    maxRunTimeSanitizerModeWallTime = 3480.0
```

Comment:
I haven't discussed this with the team at large, but it's my opinion that our normal correctness tests should be written to not exceed something like 5 minutes in duration and the sanitizer tests should not exceed 15 minutes. These timeouts don't completely cover the entire runtime of the test, so my preference would to set these to something like 250 and 800, respectively. At the very least, I think they should be tuned well below the cutoff point.

Reply:
Those make sense.  I had originally suggested around 20 minutes but in practice I can't remember a bug which did not have any reproductions under 5 minutes of run time (which becomes >5x longer with redwood debug logging enabled anyway).

Reply:
Done. Changed to 250 and 800 sec, respectively.